scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -756,6 +756,7 @@ EOF
       getRewards ${wallet_name}
       if [[ "${reward_lovelace}" -ge 0 ]]; then
         say "$(printf "%-19s : ${FG_CYAN}%s${NC} ADA" "Rewards" "$(formatLovelace ${reward_lovelace})")" "log"
+        say "$(printf "%-19s : ${FG_CYAN}%s${NC} ADA" "Funds + Rewards" "$(formatLovelace `expr ${base_lovelace} + ${reward_lovelace}`)")" "log"
```

Comment:
Best to hold this part out on specific screen until HD wallet support as it will be inaccurate regardless of what we enter as field here

Reply:
Meant to respond back to this and inquire for knowledge sake. What is the inaccuracy? Something in the way we calculate the balances I would assume since as this is just addition.

Reply:
What we should be aiming for would be "Total Delegated" and that would encompass HD Wallet balance + Rewards. Currently a Funds + Rewards gives possibly a wrong picture as your wallet could have a lot more balance on addr_ix > 1 than what is seen by CNTools